--- ghc-9.12.1/hadrian/src/Settings/Default.hs	2024-12-16 00:59:46.000000000 +0530
+++ ghc-9.12.1.mod/hadrian/src/Settings/Default.hs	2025-02-23 17:29:17.939701196 +0530
@@ -171,12 +171,12 @@
         , transformers
         , unlit
         , xhtml
+        , iserv
         , if winTarget then win32 else unix
         ]
       , when (not cross)
         [ haddock
-        , hpcBin
-        , iserv
+        , hpcBin 
         , runGhc
         , ghcToolchainBin
         ]
