--- ghc-9.12.1/hadrian/src/Settings/Default.hs	2025-02-24 22:46:43.668323779 +0530
+++ ghc-9.12.1.mod/hadrian/src/Settings/Default.hs	2025-02-26 01:15:21.522950711 +0530
@@ -116,7 +116,7 @@
              -- that confused Hadrian, so we must make those a stage0 package as well.
              -- Once we drop `Win32`/`unix` it should be possible to drop those too.
              ]
-          ++ [ terminfo | not windowsHost, not cross ]
+          ++ [ terminfo | not windowsHost ]
           ++ [ timeout  | windowsHost                ]
 
 -- | Packages built in 'Stage1' by default. You can change this in "UserSettings".
@@ -171,14 +171,15 @@
         , transformers
         , unlit
         , xhtml
+        , iserv
+        , runGhc
+        , hpcBin
+        , haddock
         , if winTarget then win32 else unix
         ]
       , when (not cross)
-        [ haddock
-        , hpcBin
-        , iserv
-        , runGhc
-        , ghcToolchainBin
+        [
+        ghcToolchainBin
         ]
       , when (winTarget && not cross)
         [ -- See Note [Hadrian's ghci-wrapper package]
